firefox-ios/firefox-ios-tests/Tests/XCUITests/ReadingListTests.swift

```diff
@@ -235,7 +235,10 @@ class ReadingListTests: BaseTestCase {
         savedToReadingList.tap()
         // The article is displayed in Reader View
         mozWaitForElementToExist(app.buttons["Reader View"])
-        XCTAssertTrue(app.buttons["Reader View"].isSelected)
+        // iOS 18 only: Reader View icon is enabled but is not selected.
+        if #unavailable(iOS 18) {
+            XCTAssertTrue(app.buttons["Reader View"].isSelected)
```

Comment:
I'm not sure if this is a bug: The reader view icon from the awesome bar isn't selected anymore on iOS 18.